UPGRADING.md

```diff
@@ -0,0 +1,105 @@
+# Upgrading
+
+This section describes how to upgrade a Conjur Server.
+
+## Standard Process
+
+The following steps describe a standard upgrade of the Conjur server, when deployed
+using Docker Compose. These steps assume you have defined your Conjur image in
+a service named `conjur`, and that you have access to the Conjur data key
+that was used when you originally deployed your Conjur server.
+1. In your terminal window, set the `CONJUR_DATA_KEY` environment variable:
+   ```
+   export CONJUR_DATA_KEY={your Conjur data key}
+   ```
+
+1. Edit the Conjur image version in `docker-compose.yml` to reference the new
+   version.
+
+1. Pull the new Conjur image version:
+   ```
+   docker-compose pull conjur
+   ```
+
+1. Stop the Conjur container:
+   ```
+   docker-compose stop conjur
+   ```
+
+1. Bring up the Conjur service using the new image version without changing
+   linked services:
+   ```
+   docker-compose up -d --no-deps conjur
+   ```
+
+1. View Docker containers and verify all are healthy, up and running:
+   ```
+   docker ps -a
+   ```
+
+   It may also be useful to check the logs of the Conjur
+   container to ensure that Puma started successfully, which can be done by
+   running
+   ```
+   docker logs conjur_server
```

Comment:
In previous steps with refered to `conjur`, now it's `conjur_server` - as a conjur developer I know that this is the default values: `conjur` service name, `conjur_server` container name. For customer it might be confusing

`docker-compose logs conjur`

Sugessting to keep using `docker-compose` as:
![image](https://user-images.githubusercontent.com/50704381/87645294-f1236e00-c755-11ea-9abb-d3a7bef149e5.png)